@asyncapi/parser doesn't work in browser

#### Describe the bug

I created a new `react` app and add in dependencies `@asyncapi/parser`. Making example as in Readme.md I see the error:

```bash
Module not found: Can't resolve './build/Release/re2'
``` 

#### How to Reproduce
Steps to reproduce the issue. Attach all resources that can help us understand the issue:

Create any browser app, but I give react example:
- `npx create-react-app my-app`
- `cd my-app && npm i @asyncapi/parser`
- change main `App.js` component to:

```js
import { parse } from "@asyncapi/parser";

function App() {
  useEffect(() => {
    const lol = async () => {
      const doc = await parse(`
        asyncapi: '2.0.0'
        info:
          title: Example
          version: '0.1.0'
        channels:
          example-channel:
            subscribe:
              message:
                payload:
                  type: object
                  properties:
                    exampleField:
                      type: string
                    exampleNumber:
                      type: number
                    exampleDate:
                      type: string
                      format: date-time
      `);
      const channels  = doc.channels();
      console.log(channels);
    }
  });

  return null;
}
```

- you should see error, which is mentioned in description of bug.

#### Expected behavior

Parser should works in node and in browser without errors.

#### Possible solution

Use native Regex, not `Re2`, because it has a compiled binary, and also investigate if `node-fetch` could problems running on browser.

crap, I went too far with sonarcube recommendations, forgot about the browser
`Re2` is not a must, but we need to make sure we mitigate a security issue

root of the issue is that bundle is not updated at all, and now it is empty in the master:
```
$ npm run prepublishOnly

> @asyncapi/parser@0.26.0 prepublishOnly /Users/wookiee/Documents/sources/parser-js
> npm run bundle


> @asyncapi/parser@0.26.0 bundle /Users/wookiee/Documents/sources/parser-js
> browserify lib/browser.js | uglifyjs > dist/bundle.js

Error: Cannot find module './build/Release/re2' from '/Users/wookiee/Documents/sources/parser-js/node_modules/re2'
    at /Users/wookiee/Documents/sources/parser-js/node_modules/browser-resolve/node_modules/resolve/lib/async.js:55:21
    at load (/Users/wookiee/Documents/sources/parser-js/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/Users/wookiee/Documents/sources/parser-js/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /Users/wookiee/Documents/sources/parser-js/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqCallback.oncomplete (fs.js:166:21)
```

---
regarding `node-fetch`, I hope browserify is smart and replaces it with native fetch for a browser (wishful thinking 😄 )

--- a/lib/utils.js
+++ b/lib/utils.js
@@ -3,7 +3,6 @@ const { yamlAST, loc } = require('@fmvilas/pseudo-yaml-ast');
 const jsonAST = require('json-to-ast');
 const jsonParseBetterErrors = require('../lib/json-parse');
 const ParserError = require('./errors/parser-error');
-const RE2 = require('re2');
 
 const jsonPointerToArray = jsonPointer => (jsonPointer || '/').split('/').splice(1);
 
@@ -264,9 +263,8 @@ utils.improveAjvErrors = (errors, asyncapiYAMLorJSON, initialFormat) => {
  */
 utils.parseUrlVariables = str => {
   if (typeof str !== 'string') return;
-  const regex = new RE2(/{(.+?)}/g);
 
-  return regex.match(str); 
+  return str.match(/{(.+?)}/g); 
 };
 
 /**

--- a/package.json
+++ b/package.json
@@ -56,7 +56,6 @@
     "js-yaml": "^3.13.1",
     "json-to-ast": "^2.1.0",
     "node-fetch": "^2.6.0",
-    "re2": "^1.15.0",
     "tiny-merge-patch": "^0.1.2"
   },
   "browserify": {

